pycontrails/core/met.py

```diff
@@ -1022,6 +1037,84 @@ def to_vector(self, transfer_attrs: bool = True) -> vector_module.GeoVectorDatas
             vector.attrs.update({str(k): v for k, v in self.attrs.items()})
         return vector
 
+    def _get_attr_template(
+        self,
+        name: str,
+        supported: tuple[str, ...],
+        examples: dict[str, str],
+    ) -> str:
+        """Look up an attribute with a custom error message."""
+        try:
+            out = self.attrs[name].upper()
```

Comment:
Why `upper()`?

Edit: OK i see, you are comparing to the pre-defined attrs. I would put an `.upper()` on supported below, so you re just making the comparison case insensitive.

Reply:
Maybe rename this `_get_pycontrails_attr_template` so that it specifies this isn't a generic `get_attr` thing?

Reply:
I removed the call to `upper()`. We don't use that sort of pattern when working with aircraft ICAO codes, so there's no reason to allow case insensitivity here. I think keeping this case sensitive will help keep everything more explicit.

c95b2b5b2ecfe111193fc8128c7a3b6386f0d880